src/utils/withPageTracker.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import ReactGA from 'react-ga';
+
+let currentPage = null;
```

Comment:
I think this is a bad idea as this could be mutated from anywhere. I would instead make this track based on the decorator if you need to override:

```js
@withPageTracker("/tasks")
```

Reply:
Probably a silly question, but how could it be mutated if it's not exported?

Reply:
You are exporting the component which exposes `trackPage`, which mutates a variable that doesn't change with each export, so every component that calls `trackPage` would all mutate the same `currentPage`.

Reply:
Worst case scenario, you can store the `currentPage` on `this`:

```js
currentPage = this;

trackPage(page) {
  this.currentPage = page;
}
```

Reply:
Aside from the use of the odious `=`, that seems like the best case. :)

Reply:
It's a class property lol.

Reply:
> Worst case scenario, you can store the currentPage on this.

I like this approach. Thanks.